jump/src/context.rs

```diff
@@ -630,11 +675,16 @@ mod tests {
             size: 42,
             version: "0.1.0".to_string(),
         };
-        let tempdir = tempfile::tempdir().unwrap();
         let lift = Lift {
             name: "test".to_string(),
             description: None,
-            base: Some(tempdir.path().to_path_buf()),
+            base: Some(
+                PathBuf::from("{scie.user.cache_dir={scie.env.USER_CACHE_DIR=/tmp/nce}}")
```

Comment:
@sureshjoshi this is the new test section to expand and study to see the placeholder in use.